bugsnag-plugin-android-ndk/src/main/jni/bugsnag_ndk.c

```diff
@@ -36,13 +36,28 @@ void bsg_release_env_write_lock(void) {
   pthread_mutex_unlock(&bsg_global_env_write_mutex);
 }
 
+/**
+ * Get the configured unwind style for non-async-safe environments.
+ * DO NOT USE THIS IN A SIGNAL HANDLER!
+ */
 bsg_unwinder bsg_configured_unwind_style() {
```

Comment:
This is only used from `bugsnag.c` to unwind for `notify()`, so maybe this function should move to be internal to that file? Otherwise, the comment would make more sense in the header doc for this function rather than the implementation.

Reply:
It makes use of the static `bsg_global_env`, which would have to be exposed if the function were placed in another file. This seemed like the lesser evil.